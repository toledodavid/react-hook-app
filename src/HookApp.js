import React from 'react';


const HookApp = () => {
  return(
    <div>
      <h1>Hello world</h1>
    </div>
  );
}

export default HookApp;