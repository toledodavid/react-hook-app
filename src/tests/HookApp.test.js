import '@testing-library/jest-dom';
import React from 'react';
import { shallow } from 'enzyme';
import HookApp from '../HookApp';


describe('Tests in <HookApp/> component', () => {

  test('It should render correnctly', () => {
    const wrapper = shallow(<HookApp />);
    expect(wrapper).toMatchSnapshot();
  });

});