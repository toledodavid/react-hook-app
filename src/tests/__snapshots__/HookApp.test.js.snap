// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests in <HookApp/> component It should render correnctly 1`] = `
<div>
  <h1>
    Hello world
  </h1>
</div>
`;
